Source/HotwireConfig.swift

```diff
@@ -78,11 +78,3 @@ public struct HotwireConfig {
         return configuration
     }
 }
-
-public extension HotwireConfig {
-    class PathConfiguration {
-        /// Enable to include the query string (in addition to the path) when applying rules.
-        /// Disable to only consider the path when applying rules.
-        public var matchQueryStrings = true
```

Comment:
Why did this move out of the configuration? How can I access it if the new variable isn't `static`?

I'd vote to keep this inside of the global configuration if possible.

Reply:
Disregard my comment above. The diff doesn't show that `Hotwire.config.pathConfiguration` still exists. The location of `matchQueryStrings` just moved.